### ISSUE TITLE
LBP fiat currency addition 

### ISSUE THREAD

**Issue by marcondy** [ACINQ/phoenix#344]
It would be great to have integrated the real market rate of LBP ( Lebanese Lira ) 

[API documentation ](https://yadio.io/api.html) 

https://api.yadio.io/convert/1/BTC/LBP

Thank you in advance.

**Comment by dpad85** [comment]
Good idea and thanks for the API endpoint, it seems there's a large discrepancy with the rate we use. I don't think we'll be able to deliver this quickly though.

**Comment by marcondy** [comment]
> Good idea and thanks for the API endpoint, it seems there's a large discrepancy with the rate we use. I don't think we'll be able to deliver this quickly though.

Thank you for your prompt response, no worries as long as it's going to be correct eventually. 

Majority of converters online use the "official" rate , therefore it's been wrong for over three years now.

Thank you again

**Comment by robbiehanson** [comment]
Is there a "common name" for the free market rate (to differentiate it from the official rate) ?

For example, in Argentina it's commonly called the "blue market" (or "dólar blue" or "mercado azul"). So we named the currency "Argentine Peso (blue market)" - ARSbm

When we added support for Cuban Pesos I wasn't aware of any such "common name", so we simply called it "Cuban Pesos (free market)" - CUPfm

We could do the same here: "Lebanese Pound (free market)" - LBPfm

Unless there's a common name already in circulation? Let me know your thoughts.
